static/js/main.js

```diff
@@ -240,13 +239,35 @@ function addScreenshot(screenshot) {
                             data-screenshot='${screenshot.id}'
                             value='${time_input_value}' />
                 </div>
+                <div>
+                    <button onclick='flipImage(this, "${screenshot.id}")'>Flip Image</button>
+                </div>   
             </div>
         </div>
     `;
 
     document.querySelector('.screenshot-area').insertAdjacentHTML('beforeend', html);
 }
 
+// Flip an image in the screenshot area by adding it the the canvas flipped horizontally.
+function flipImage(element, screenshot_id) {
+    // Find correct image, add to canvas to flip.
+    let img = element.parentElement.parentElement.previousElementSibling.firstElementChild;
+    let canvas = document.createElement('canvas');
+    canvas.width = img.naturalWidth;
+    canvas.height = img.naturalHeight;
+    let context = canvas.getContext('2d');
+    context.scale(-1, 1)
+    context.drawImage(img, -img.naturalWidth, 0);
+
+    // Draw the canvas to the image area.
+    img.src = canvas.toDataURL('image/jpeg');
+    
+    // Update image metadata and store flipped image.
+    window.shark.screenshots[screenshot_id]['flipped'] === false ? window.shark.screenshots[screenshot_id]['flipped'] = true : window.shark.screenshots[screenshot_id]['flipped'] = false
```

Comment:
I'm generally not a fan of ternary statements (but expressions are fine).
They can easily get very confusing and hard to debug.

I think you can simplify this to something like:
```
window.shark.screenshots[screenshot_id]['flipped'] = !Boolean(window.shark.screenshots[screenshot_id]['flipped']);
```

Reply:
Oh yeah, smart.